### ISSUE TITLE
(iOS) How to add easily a padding in scrollView content area ( using sectionInset property & UIEdgeInsets ) ?

### ISSUE THREAD

**Issue by GojiDev** [nativescript-community/ui-collectionview#30]
Hi, I have a question

I wanted to add an extra padding in the "scrollview" and i found only this function with `UIEdgeInsets`: 

```
// src/collectionview/collectionview.ios.ts > class CollectionView

    private _setPadding(newPadding: { top?: number; right?: number; bottom?: number; left?: number }) {
        const layout = this._layout;
        if (layout.hasOwnProperty('sectionInset')) {
            const padding = {
                top: layout['sectionInset'].top,
                right: layout['sectionInset'].right,
                bottom: layout['sectionInset'].bottom,
                left: layout['sectionInset'].left
            };
            // tslint:disable-next-line:prefer-object-spread
            const newValue = Object.assign(padding, newPadding);
            layout['sectionInset'] = UIEdgeInsetsFromString(`{${newValue.top},${newValue.left},${newValue.bottom},${newValue.right}}`);
        }
    }
```

I dont found in documentation (or comments) how to set "sectionInset" property of `CollectionView` layout from DOM ( I'm using Nativescript-Vue) so i have modified the library and added properties inset[Left|Top|Bottom|Right] in code and i set this property in layout when creating the view and it's works.

Now i can do this for example: ( NativeScript-Vue ) 
```
    <CollectionView
        :items="myItems"
        itemIdGenerator="id"
        insetBottom="40"
        :colWidth="33.3%">
        <v-template>
            <Image :src="item.url" />
        </v-template>
    </CollectionView>

```

### **Can I do otherwise without modifying the library?**

_( I must add a padding in scrollview to keep items above floating button when you scroll at the end )_

**NativeScript:** _7+_
**Framework:** _Vue_
**@nativescript-community/ui-collectionview version:** _v4.0.37_

**Comment by farfromrefug** [comment]
@GojiDev thanks for reporting this! I just released 4.0.38 which adds full support for padding on iOS/Android. so simply use padding properties like you would on any other view

**Comment by GojiDev** [comment]
Thanks @farfromrefug, it's perfect 😃
